Sources/TuistSupportTesting/Extensions/XCTestCase+Extras.swift

```diff
@@ -13,6 +13,10 @@ public extension XCTestCase {
 
     // MARK: - XCTAssertions
 
+    func XCTAssertEmpty<T: Collection>(_ collection: T, file: StaticString = #file, line: UInt = #line) {
+        XCTAssertEqual(collection.count, 0, "Expected to be empty but it has \(collection.count) elements", file: file, line: line)
```

Comment:
Would an `XCTAssertTrue(collection.isEmpty)` at the call site be suffice instead of the custom assert?

Reply:
I’d keep `XCTAssertEmpty` is more concise and descriptive. This is influenced by Rails and its many APIs that makes writing code so enjoyable. You write as you think:

Ensure that X is empty